CHANGELOG.md

```diff
@@ -33,6 +33,9 @@ Breaking Changes:
   - Changed `IComponentModelBuilder.BuildModel` to use `Arguments` instead of `IDictionary`
   - Changed `ILazyComponentLoader.Load` to use `Arguments` instead of `IDictionary`
 
+Bugfixes:
+- Fixed first-chance HandlerException for optional parameters (@jnm2, #450)
```

Comment:
@jnm2 You've added this inside the 5.0.0-beta001 section which has been released.